control-plane/api-gateway/controllers/gateway_controller.go

```diff
@@ -73,7 +73,7 @@ func (r *GatewayController) Reconcile(ctx context.Context, req ctrl.Request) (ct
 	var gateway gwv1beta1.Gateway
 
 	log := r.Log.WithValues("gateway", req.NamespacedName)
```

Comment:
Does `V(1)` return a logger? If so, would we just be able to do that here? I know that `Error` calls essentially ignore the log level, and I'd imagine pretty much everything that is info should be less verbose.

Reply:
Yes. Ended up doing this for the gateway controller, as all of the info statements there really don't need to be printed out unless someone is debugging. 
I had to do the gatewayClass and gatewayClassConfig controllers a little differently, so that the two special info statements (linked in description of PR) would still be printed out. There might be a reason to make them truly error statements?